src/components/tooltip/HoverTooltip.js

```diff
@@ -80,7 +79,9 @@ export default class HoverTooltip extends PureComponent {
       <div className={classes[`wrapper${inline ? 'Inline' : ''}`]}>
         <span
           onMouseOver={this.onMouseOver}
+          onFocus={this.onMouseOver}
           onMouseOut={this.onMouseOut}
+          onBlur={this.onMouseOut}
```

Comment:
Why this change?

Reply:
I added onBlur and onFocus in order to remove the error about https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/mouse-events-have-key-events.md.